src/python-common/ceph/deployment/service_spec.py

```diff
@@ -304,15 +304,17 @@ class ServiceSpec(object):
     """
 
     def __init__(self,
-                 service_type,  # type: str
-                 service_id=None,  # type: Optional[str]
+                 service_type: str,
+                 service_id: str = None,
                  placement: Optional[PlacementSpec] = None,
-                 count: Optional[int] = None):
+                 count: Optional[int] = None,
+                 unmanaged: bool = False):
```

Comment:
this change makes it painful to have python-common usable by ceph-volume.

Reply:
the new arg, or the py3 syntax?

Reply:
the new syntax for adding type annotations

Reply:
fixed